react/src/reactTextMask.js

```diff
@@ -42,15 +42,14 @@ export default class MaskedInput extends React.Component {
     delete props.onChange
     delete props.showMask
 
-    return (
-      <input
-        {...props}
-        onBlur={this.onBlur}
-        onChange={this.onChange}
-        defaultValue={this.props.value}
-        ref={(inputElement) => (this.inputElement = inputElement)}
-      />
-    )
+    const ref = (inputElement) => (this.inputElement = inputElement)
+
+    return render(ref, {
+      onBlur: this.onBlur,
+      onChange: this.onChange,
+      defaultValue: this.props.value,
+      ...props
```

Comment:
Maybe add a trailing comma for cleaner diffs?

Reply:
Agree, this is a good convention to follow - updated.  Although worth mentioning that any props listed after the spread `...props` will overwrite - so I doubt we'd ever want to have anything trailing this value.